sandbox/example/node_modules/@loopback/tslint-config

```diff
@@ -0,0 +1 @@
+../../../../packages/tslint-config
```

Comment:
I think this file does not belong to this pull request, could you remove it please?